src/System Application/App/File System/README.md

```diff
@@ -0,0 +1,7 @@
+# File System Module for Business Central
```

Comment:
During manual testing, I still sometimes cannot find the files I uploaded in the browser. If I go to the Azure Storage Browser, the files are visible, but not from within Business Central. We must eliminate that bug, before release. Happy to do a screensharing session to reproduce. It's straight forward:

With Azure File Storage:
Create a folder structure more than 1 level deep (mine was 2)
Put files in the 2nd level
Expected: the file shows in browser
Actual: Nothing shows.

Reply:
Azure File Share or Blob? I was able to reproduce it wit ahzure file share and fixed it.
Can you try it again?

Reply:
I for sure will try again!